6.5/cdm/README.md

```diff
@@ -22,13 +22,20 @@ to such license between the licensee and ForgeRock AS.
 - Single realm
 - OAuth 2.0 Server and Client
 - Separate userstore, configstore and ctsstore
+- CTS with Affinity across two servers
 - LDAP Auth module default
-- To log into admin console use https://fqdn/XUI/?service=adminconsoleservice
+- Stateful sessions and stateful OAuth 2.0
+- All connection pools tuned
+- Using DS TTL for token reaping and the new GrantSet Token Storage Scheme
+	- Refer to the product documents on some limitions on notifications caused by using DS TTL
+- To log into admin console specify the "adminconsoleservice" as query parameter.  For example
+	- https://login.example.com/XUI/?service=adminconsoleservice
 
 ### IDM Config
 - PostgreSQL as repo
-- Syncronization to LDAP (userstore)
-- This sample demonstrates bidirectional synchronization between an LDAP directory and an OpenIDM repository. For documentation relating to this sample, see https://backstage.forgerock.com/docs/idm/samples-guide/index.html#chap-sync-with-ldap-bidirectional
+- Synchronization to LDAP (userstore)
+- This sample demonstrates bidirectional synchronization between an LDAP directory and an OpenIDM repository. 
+	- For documentation relating to this sample, see https://backstage.forgerock.com/docs/idm/samples-guide/index.html#chap-sync-with-ldap-bidirectional
```

Comment:
For completeness, would it be possible to add a little bit about the IG configuration, too?

Reply:
NVM, I see it's in there, just didn't see it in the diffs. Apologies.

Reply:
Thanks for updating. This is an important README for us. We link to it from here: https://ea.forgerock.com/docs/platform/gke-cookbook/index.html#cookbook-using-cdm-components